M2/Macaulay2/packages/Normaliz.m2

```diff
@@ -288,10 +292,10 @@ doWriteNmzData(List):=(matrices)->
      if nmzModes#?nmzMode then(
         -- deprecated input type as integer
         print("Using Normaliz integer input types is deprecated, please use " | nmzModes#nmzMode | " instead of " | nmzMode);
-        outf << nmzModes#nmzMode << endl;
-     )
-     else
-        outf << nmzMode << endl;
+        nmzMode = nmzModes#nmzMode);
+     -- Until version 3.9.4, input type normal_toric_ideal was called lattice_ideal
+     if normalizProgram#"version" < "3.10" and nmzMode == "normal_toric_ideal" then nmzMode = "lattice_ideal";
```

Comment:
I *think* this answers the previous question about normaliz...

Reply:
This PR is really on top of PR https://github.com/Macaulay2/M2/pull/2763 which I made to fix this, but I think @DanGrayson hasn't looked at it. The nightly tests on macOS have been failing for over a week.